pages/linux/rnm.md

```diff
@@ -0,0 +1,37 @@
+# rnm
+
+> Bulk Rename Utility.
+> A powerful batch renaming tool that supports search/replace, indexing, sorting, undo, and more.
+> More information: <https://github.com/neurobin/rnm>.
+
+- Replace a search string with a replacement string in filenames:
+
+`rnm -ss {{old}} -rs {{new}} {{path/to/files}}`
+
+- Use a fixed (literal) search and replace string instead of regex:
+
+`rnm -ssf {{old}} -rs {{new}} {{path/to/files}}`
+
+- Add an auto-incremented index to filenames starting from 1:
+
+`rnm -i 1 -inc 1 -rs {{_}} {{path/to/files}}`
+
+- Rename files using a list of new names from a text file:
+
+`rnm -ns/f {{path/to/names.txt}} {{path/to/files}}`
```

Comment:
Is `-ns/f` part of the syntax or is this meant to be interpretted?

Reply:
It's part of it.  
`-ns` is for the new name(s) string. 
`/f` can be added to get the name(s) from a file instead.